### DISCUSSION TITLE
Similar problem to 3382, Strong name signature could not be verified [but this time] on Microsoft.Extensions.VectorData.Abstractions

### DISCUSSION THREAD

**Discussion by Reboot4X** [microsoft/semantic-kernel#9329] [General]
I'm seeing this runtime error, which is similar to the problem in #34382 ( https://github.com/microsoft/semantic-kernel/issues/3382) but for Microsoft.Extensions.VectorData.Abstractions (9.0.0-preview.1.24518.1)

The project is a .net framework 4.8 one that just implements a simple web forms app with the code from this tutorial:  
https://learn.microsoft.com/en-us/dotnet/ai/quickstarts/quickstart-local-ai

The Error I get when running the application: 

Could not load file or assembly 'Microsoft.Extensions.VectorData.Abstractions' or one of its dependencies. Strong name signature could not be verified.  The assembly may have been tampered with, or it was delay signed but not fully signed with the correct private key. 

=== Pre-bind state information ===
LOG: DisplayName = Microsoft.Extensions.VectorData.Abstractions
 (Partial)
WRN: Partial binding information was supplied for an assembly:
WRN: Assembly Name: Microsoft.Extensions.VectorData.Abstractions | Domain ID: 2
WRN: A partial bind occurs when only part of the assembly display name is provided.
WRN: This might result in the binder loading an incorrect assembly.
WRN: It is recommended to provide a fully specified textual identity for the assembly,
WRN: that consists of the simple name, version, culture, and public key token.

----

So I tried the steps from issue 3382 and pulled out the dll from the  ....\package\microsoft.extensions.vectordata.abstractions.9.0.0-preview.1.24518.1.zip\lib\netstandard2.0 directory and replaced the dll in my bin directory and that gets rid of the error.   (this one under netstandard2.0 is signed)

Not sure why the signed dll isn't the one that build copied to the bin directory.    Is this an issue that should be logged or some configuration that I need to work through on my system?

**Comment by Reboot4X** [comment]
The steps from 3382 are a good test, but seem a bit fragile for a long term solution as the dll could get copied with the unsigned one....

**Comment by dmytrostruk** [comment]
@Reboot4X Thanks for reporting this issue! We added .NET Framework version of `Microsoft.Extensions.VectorData.Abstractions` package to the signing process, so the package should be signed after next release. In case you will see the same issue after next release, please let us know. Thanks again!
